src/SUMMARY.md

```diff
@@ -1,5 +1,15 @@
 # Summary
 
-- [Onion client](./onion_client.md)
-- [Messenger packets](./messenger_packets.md)
-- [Conference packets](./conference_packets.md)
+[Introduction](intro.md)
+
+- [Cryptography](crypto/crypto.md)
+- [Network](network/network.md)
```

Comment:
I'd like to move all chapters with "TODO" into the bottom, because otherwise people will have to scroll 6 chapters to find a chapter with an info

Reply:
This would mess the order of the chapters. It is not random, it goes from low-level to high-level.

Reply:
OK. What will be Network about?

Reply:
About, well, network. Like packet structure overview, etc.

Reply:
We use TCP and UDP so there will be no packet structure overview for Network chapter

Reply:
You want at least a list of packet types somewhere. Like in [here](https://toktok.ltd/spec.html#protocol-packet).

Reply:
It is useless.

Reply:
For a good book it is definitely not useless. We are going to have a book instead of a pile of dev notes where syntax is mashuped with semantics, are we?

Reply:
It is useless because neither TCP handshake meets these conditions nor https://github.com/tox-rs/tox/blob/master/src/toxcore/tcp/packet/data.rs, there are exceptions.

https://toktok.ltd/spec.html#protocol-packet this chapter describes DHT packets which is another story. We will copy that, but it does not mean we have to make a special top level chapter.